pkg/version/windows_versioninfo.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
We have a function `os::build::get_product_vars` in `hack/lib/build/binaries.sh` that holds things we expect to change between Origin and OCP -- the name is not in there yet but we can definitely add it.